english/count-the-marshmellows/eat-all-the-marshmellows/get-breakfast/get-breakfast.md

```diff
@@ -1,7 +1,14 @@
-HECK YES. You found the best ending. (Breakfast is always the best ending.)
```

Comment:
I pasted the entire paragraph back in but it keeps looking like I deleted it, and I don't know how to fix it.